docs/SENTENCES.md

```diff
@@ -34,8 +41,21 @@ Feel free to set up this QA however makes most sense for you, but here's a [samp
 
 Once the review is complete, submit a pull request with the # of sentences submitted, a link to the manual QA results, and the % error rate. Here's [an example PR](https://github.com/mozilla/common-voice/pull/2873). Please make sure the sentences are in a plain `.txt` file with one sentence per line.
 
-## Flagging problematic sentences
+## Rules for QA that applies (or not) to the different imputs and outputs
+To help you, for exemple with the 'not supervised, do your own QA' bulk submission, you _may_ find interesting to have a look at : 
```

Comment:
While this may be helpful, it is not a rule for QA of bulk submissions, so I'm a bit hesitant in listing this here as is. @ftyers do you have thoughts on this?

Reply:
Indeed, QA is requested... But not "how to do it"! you're right. 

Here is the reworked version : 

## QA that applies (or not) to the different imputs and outputs
As a reminder, here is listed the different automated QA applied, depending of the process.
* [cleanup](https://github.com/common-voice/sentence-collector/tree/main/server/lib/cleanup) and [validation](https://github.com/common-voice/sentence-collector/tree/main/server/lib/validation) rules for Sentence Collector, 
* [language](https://github.com/common-voice/cv-sentence-extractor#using-language-rules) rules for Sentence Extractor, 
* [preprocessor](https://github.com/common-voice/CorporaCreator/tree/master/src/corporacreator/preprocessors) for CorporaCreator ([CorporaCreator](https://github.com/common-voice/CorporaCreator) is the Common Voice extractor that, linking text and recorded voice samples, make them available for [SST](https://en.wikipedia.org/wiki/Speech_recognition) engines' training).

To help you, for exemple with the 'not supervised, do your own QA' bulk submission, you may find interesting to have a look at it.